src/neo/SmartContract/Native/NativeContract.cs

```diff
@@ -26,12 +27,32 @@ public abstract class NativeContract
         public static OracleContract Oracle { get; } = new OracleContract();
         public static DesignationContract Designation { get; } = new DesignationContract();
 
+        static NativeContract()
```

Comment:
I think this might be problematic, as static constructor will be executed only once upon a native contract constructed at the first time

Reply:
Fixed, please take a look again @Qiao-Jin